docs/HOWTO -- NVIDIA AGX ORIN  on-device kernel build.md

```diff
@@ -0,0 +1,112 @@
+# HOWTO -- NVIDIA AGX ORIN on kernel on-device development process
+
+
+Nvidia is powerful device so developing and something lowlevel as drivers and passthroughs may be fast and efficient when done on board directly.
+
+
+1. Preparational steps -- get a device, plug in monitor and keyboard (in most cases you need DP cable and monitor with DP socket, adapters DP-HDMI will not work before firmware update).
+2. Boot and update onboard L4T ubuntu based linux distribution -- here are the link to the nvidia instructions -- .
+3. Get a M.2 SSD nvme drive ( 512Gb or more is recommended ) and install it in the slot on the bottom of the device.
```

Comment:
What does it mean particularly "get"? Download? Do we have some official site?

Anyway, it should be rewritten like this:

3. Get a M.2 SSD NVMe drive (512GB or more is recommended -- should we take it to the requirements???) and install it in the slot on the bottom of the device.